sockeye/lr_scheduler.py

```diff
@@ -29,8 +29,11 @@ def __init__(self, warmup: int = 0) -> None:
         self.log_warmup_every_t = self.warmup // 10
         self.last_warmup_log = -1
 
-    def new_evaluation_result(self, has_improved: bool):
-        pass
+    def new_evaluation_result(self, has_improved: bool) -> bool:
+        """
+        Returns true if learning rate was adjusted.
```

Comment:
looks good, but the naming is a little confusing as there are different lr scheduler that constantly adjust the learning rate, but will never return True. Why not make it explicit:
```
Returns true when the parameters should be reset to the ones with the best validation score.
```

Reply:
Good idea, done